R/pkg/R/DataFrame.R

```diff
@@ -244,11 +245,15 @@ setMethod("showDF",
 #' @note show(SparkDataFrame) since 1.4.0
 setMethod("show", "SparkDataFrame",
           function(object) {
-            cols <- lapply(dtypes(object), function(l) {
-              paste(l, collapse = ":")
-            })
-            s <- paste(cols, collapse = ", ")
-            cat(paste(class(object), "[", s, "]\n", sep = ""))
+            if (identical(sparkR.conf("spark.sql.repl.eagerEval.enabled", "false")[[1]], "true")) {
```

Comment:
`show` also works on other Spark object like `Column`, I think we only need to do `showDF` for `SparkDataFrame`?

Reply:
Or the param doc is not correct?

Reply:
I think the `param` is not correct... the signature of `show()` has `SparkDataFrame` so in this one it won't work for `Column`. In column.R file, it has its own `show()` implementation.

Reply:
Can you also correct the param doc?

Reply:
I might have misunderstood your previous question. The `@param` does not need to change as it  is common to `SparkDataFrame` and `Column` etc for `show()` method. Same applies to description, so I rewrote the description. Please have a look.

Reply:
^ that - it's two different "overloads" for two different "class"